app/src/main/res/values/strings.xml

```diff
@@ -87,25 +87,25 @@
     <string name="action_show_solution">Show solution</string>
     <string name="action_show_mine_sudoku">Show mine</string>
     <string name="onboard_title">Welcome</string>
-    <string name="intro_what_is_sudoku">A classic sudoku looks as follows</string>
-    <string name="intro_rules">Use 1–9 to fill all cells—without repetition in any column, row or smaller (3x3) box.</string>
-    <string name="onboard_recommendation_prefs">Go through the settings to adjust the game</string>
+    <string name="intro_what_is_sudoku">A classic sudoku</string>
+    <string name="intro_rules">Fill cells using 1–9 without repetition in any column, row, or smaller (3x3) box.</string>
+    <string name="onboard_recommendation_prefs">Adjust the settings to your liking.</string>
     <string name="action_start">Start</string>
-    <string name="toolbar_undo_description">Undo. Just undo</string>
-    <string name="toolbar_notes_description">Activates the note mode (pencil marks)</string>
-    <string name="toolbar_hint_description">Solves the selected cell. Can be turned off in settings</string>
-    <string name="toolbar_erase_description">Removes a number or all notes from the selected cell</string>
+    <string name="toolbar_undo_description">Undo. Just undo.</string>
+    <string name="toolbar_notes_description">Note mode (pencil marks)</string>
+    <string name="toolbar_hint_description">Solves the selected cell. (Can be turned off.)</string>
```

Comment:
I would remove that sentence entirely. "Solve selected cell." makes sense to me.
I agree it looks out of context, but then again maybe that is because it is.
Semantically it is correct.
https://www.grammarbook.com/punctuation/parens.asp
Periods go inside parentheses only if an entire sentence is inside the parentheses.